apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Trace/CloudTraceExtension.cs

```diff
@@ -121,7 +124,15 @@ public static void UseGoogleTrace(this IApplicationBuilder app)
         /// </summary>
```

Comment:
I don't think so I think we still need the IManagedTracer to be scoped right?  Unless we are going to return Func<IManagedTracer> instead of IManagedTracer?